quattor/types/hardware.pan

```diff
@@ -186,6 +185,21 @@ type structure_hardware = {
     "console"      ? structure_console
     "sysloc"       ? structure_sysloc
     "nodename"     ? string
+<<<<<<< HEAD
+    # benchmarks is used to hold the performance benchmark for the machine 
+    # i.e. HEPSpec06 score
+    # this might be used to scale things such as the wall time
+    # example of using this might be : 
+    # variable CONDOR_WN_SCALING_FACTOR = value('/hardware/benchmark/HEPSpec06') 
+=======
+    #benchmarks is used to hold the performance benchmark for the machine 
+    #i.e. HEPSpec06 score
+    #this might be used to scale things such as the wall time
+    #example of using this might be : 
+    #variable CONDOR_WN_SCALING_FACTOR = value('/hardware/benchmark/HEPSpec06') 
+>>>>>>> origin/master
```

Comment:
Conflict leftovers... Do **not** merge master into your topic branches.  The workflow is topic -> master, never the other way around.